README.md

```diff
@@ -1,33 +1,63 @@
-# Windows-on-arm64 GNU toolchain cross compiler.
+# Windows on Arm64 GNU Toolchain Cross-Compiler
 
-[![Build toolchain](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml)
+[![Build main toolchain](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml) [![Build toolchain variants](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/advanced.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/advanced.yml) [![Daily rebase](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/rebase.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/rebase.yml)
 
-This repository contains a script to build a GNU toolchain cross compiler for Windows-on-arm64. It is a **work in progress**. This allows binaries that can execute on Windows-on-arm64 to be built on Linux. I have only tested this script on Ubuntu/WSL.
+This repository contains scripts for building a `x86_64-pc-linux-gnu` host, `aarch64-w64-mingw32` target GNU toolchain cross-compiler for Windows on Arm64. It is a **work in progress** in the stage of upstreaming the necessary changes to the corresponding repositories. This toolchain contains binaries that can be executed on Windows on Arm64 to be built on Linux. The scripts are being actively tested on the default Ubuntu 22.04 GitHub Actions runners and Ubuntu 22.04 in WSL.
```

Comment:
I would rephrase it: for building the aarch64-w64-mingw32 target on aarch64-pc-linux-gnu or x86_64-pc-linux-gnu hosts.

Reply:
Rephrased.